administrator/components/com_admin/sql/updates/postgresql/5.1.1-2024-04-18.sql

```diff
@@ -0,0 +1,10 @@
+--
+-- Add a default value for the colorScheme in the Atum tenmplate on Joomla update
+-- only when a value is not already set.
+-- New installs will have the default value set in the installation sql.
+--
+
+UPDATE "#__template_styles"
+SET "params" = jsonb_set("params", '{colorScheme}', '"os"', true)
```

Comment:
```suggestion
SET "params" = jsonb_set("params"::jsonb, '{colorScheme}', '"os"', true)
```
We should explicitly cast the "params" column which is of type "text" to type "jsonb" so we don't rely on implicit type casting.